qa-tests-backend/src/test/groovy/ImageSignatureVerificationTest.groovy

```diff
@@ -308,12 +332,35 @@ nzTe7BpOmVwmqLkIefEJe5L4PSXtp2KFLZqGO/kY5A==
 
         // Signature integartion "BYOPKI-Wildcard+Tekton" which holds both BYOPKI wildcard and Tekton.
         String byopkiWildcardAndTektonSignatureIntegrationID = createSignatureIntegration(
-                BYOPKI_WILDCARD_AND_TEKTON, TEKTON_COSIGN_PUBLIC_KEY, BYOPKI_ROOT_CA, BYOPKI_WILDCARD_IDENTITY,
-                BYOPKI_WILDCARD_ISSUER
+            BYOPKI_WILDCARD_AND_TEKTON, TEKTON_COSIGN_PUBLIC_KEY,
+            [chain: BYOPKI_ROOT_CA, identity: BYOPKI_WILDCARD_IDENTITY, issuer: BYOPKI_WILDCARD_ISSUER]
         )
         assert byopkiWildcardAndTektonSignatureIntegrationID
-        CREATED_SIGNATURE_INTEGRATIONS.put(BYOPKI_WILDCARD_AND_TEKTON,
-                byopkiWildcardAndTektonSignatureIntegrationID)
+        CREATED_SIGNATURE_INTEGRATIONS.put(BYOPKI_WILDCARD_AND_TEKTON, byopkiWildcardAndTektonSignatureIntegrationID)
+
+        // Signature integration "Keyless-Sigstore-Matching" which holds the default Sigstore CAs
+        // and enables transparency log validation.
+        String keylessSigstoreMatchingSignatureIntegrationID = createSignatureIntegration(
+            KEYLESS_SIGSTORE_MATCHING, [:],
+            [chain: "", identity: KEYLESS_SIGSTORE_IDENTITY, issuer: KEYLESS_SIGSTORE_ISSUER,
+            ctlogEnabled: true],
+            [enabled: true, url: "https://rekor.sigstore.dev"]
+        )
+        assert keylessSigstoreMatchingSignatureIntegrationID
+        CREATED_SIGNATURE_INTEGRATIONS.put(KEYLESS_SIGSTORE_MATCHING, keylessSigstoreMatchingSignatureIntegrationID)
+
+        // Signature integration "Keyless-Sigstore-Unverifiable" which holds the default Sigstore CAs
+        // and disables transparency log validation. Verification must fail because the certificate issued
+        // by Fulcio has expired and can only be verified with the timestamp from the transparency log entry.
+        String keylessSigstoreUnverifiableSignatureIntegrationID = createSignatureIntegration(
+            KEYLESS_SIGSTORE_UNVERIFIABLE, [:],
+            [chain: "", identity: KEYLESS_SIGSTORE_IDENTITY, issuer: KEYLESS_SIGSTORE_ISSUER,
+            ctlogEnabled: true],
+            [enabled: false]
+        )
```

Comment:
I think it will be good to use explicit types instead of implicit cast from map
https://groovy-lang.org/objectorientation.html#_named_parameters

```suggestion
        String keylessSigstoreUnverifiableSignatureIntegrationID = createSignatureIntegration(
                KEYLESS_SIGSTORE_UNVERIFIABLE, [:],
                CertificateVerificationArgs.newInstance(chain: "",
                        identity: KEYLESS_SIGSTORE_IDENTITY,
                        issuer: KEYLESS_SIGSTORE_ISSUER,
                        ctlogEnabled: true),
                TransparencyLogVerificationArgs.newInstance(enabled: false, url: "")
        )
```
or better create dedicated constructor